web/src/flow/stages/identification/IdentificationStage.ts

```diff
@@ -173,20 +173,37 @@ export class IdentificationStage extends BaseStage<
                     if (!this.host) return;
                     this.host.challenge = source.challenge;
                 }}
-                class=${this.challenge.showSourceLabels ? "pf-c-button pf-m-link" : ""}
+                class=${this.challenge.showSourceLabels ? "pf-c-button pf-m-link source-buttons" : "source-buttons"
+            } style="padding: 10px 20px; border: 2px solid #0078d4; border-radius: 5px; display: flex; align-items: center; position: relative; transition: background-color 0.3s ease;"
+            onmouseover="this.style.backgroundColor='#0078d4'; this.style.color='#fff';"
+            onmouseout="this.style.backgroundColor=''; this.style.color='';"
```

Comment:
This doesn't comply with the current visual standard; we'd much prefer if you stuck with the buttons we already have. I know, we _want_ nicer buttons, but it's better to have uniform ultilitarian buttons than a few standout "awesome" ones amongst the utilitarian ones.  It confuses users by implying a different affordance or behavior.